src/analyses/libraryDesc.ml

```diff
@@ -122,6 +123,7 @@ let special_of_old classify_name = fun args ->
   | `Malloc e -> Malloc e
   | `Calloc (count, size) -> Calloc { count; size; }
   | `Realloc (ptr, size) -> Realloc { ptr; size; }
+  | `Free ptr -> Free ptr
```

Comment:
Is this needed at all? Is special_of_old ever called for `Free ?

Reply:
Seems like it's not needed. I'll make sure to remove it

Reply:
I think that I forgot to remove the `` `Free `` constructor of the `categories` type in `libraryFunctions.ml` (cf. https://github.com/mrstanb/analyzer/blob/uaf-analysis-first-iteration/src/analyses/libraryFunctions.ml#L458). I'm assuming `` `Free `` is not needed there either, right?